templates/repo/issue/view_content/comments.tmpl

```diff
@@ -314,7 +316,7 @@
 				{{template "shared/user/avatarlink" dict "Context" $.Context "user" .Poster}}
 				<span class="text grey muted-links">
 					{{template "shared/user/authorlink" .Poster}}
-					{{$.locale.Tr "repo.issues.due_date_remove" .Content $createdStr | Safe}}
+					{{$.locale.Tr "repo.issues.due_date_remove" (DateTime "long" .Content .Content) $createdStr | Safe}}
```

Comment:
It is `short` format in database?

Reply:
long means full month name (e.g. April), short means short month name (e.g. Apr)